ui/src/router/index.js

```diff
@@ -212,7 +212,7 @@ router.beforeEach((to, from, next) => {
       store.dispatch("user/setUserData", {});
       store.dispatch("user/setToken", "");
       store.dispatch("user/setItems", []);
-      Cookies.remove("user");
+      Cookies.remove("access_token_cookie");
 
```

Comment:
The Cookies.remove here is removing the cookie set by FE, is that right? Because I assume BE will not set any cookies. Also, why is the cookie name changed from "user" to "access_token_cookie"?

Reply:
Nevermind, I understand now that this is for removing the cookie if we set access_token_cookie on LoginView. Resolved.